awsrds/rds_db_instance_test.go

```diff
@@ -610,10 +722,10 @@ var _ = Describe("RDS DB Instance", func() {
 			addTagsToResourceInput *rds.AddTagsToResourceInput
 			addTagsToResourceError error
 
-			user         *iam.User
-			getUserInput *iam.GetUserInput
-			getUserError error
+			getCallerIdentityInput *sts.GetCallerIdentityInput
+			getCallerIdentityError error
 		)
+		account := "123456789012"
```

Comment:
It's not good to assign this in this way. Written like this, it only gets initialised once, and if any test modifies it, that change will persist to other tests.

If this value never changes, then we should define it as a `const`. Otherwise, it should be defined like the other variables here (with the variable declared in the `var` block above, and initialized in the `BeforeEach`).